asf_welsh_energy_consultation/getters/get_data.py

```diff
@@ -64,6 +69,20 @@ def create_argparser():
         type=str,
     )
 
+    parser.add_argument(
+        "--gold_mcs_epc_batch",
+        help="Specifies which gold merged EPC-MCS_installation-MCS_installer data batch to use. Only date required in YYMMDD format.",
+        type=str,
+    )
+
+    parser.add_argument(
+        "--download_gold_data_from_s3",
+        help="If set to True, downloads specified batch of gold merged EPC-MCS_installation-MCS_installer data from S3 locally. "
+        "Note that this download can take 30 minutes and not recommended if `hp_installed_gold_[YYMMDD]` already in supplementary data folder in `inputs`.",
+        default=False,
+        type=str,
+    )
+
```

Comment:
don't forget to update the README with these arguments and to specify which batch of gold data you used for the October analysis (and that it wasn't used in previous analyses).

Reply:
Updated